src/families/bitcoin/wallet-btc/crypto/zec.ts

```diff
@@ -74,7 +71,20 @@ class ZCash extends Base {
 
   // eslint-disable-next-line class-methods-use-this
   validateAddress(address: string): boolean {
-    return zec.Address.isValid(address, "livenet");
+    try {
+      const version = Number(
+        "0x" + bs58check.decode(address).slice(0, 2).toString("hex")
```

Comment:
You can add a comment to document this

Reply:
fixed

Reply:
where? :smile:

Reply:
why is there something specific to zcash on the "performance" topic? how should QA test this? thanks

Reply:
why is there something specific to zcash on the "performance" topic? how should QA test this? thanks